charts/minio/templates/containers/minio.yaml

```diff
@@ -2,9 +2,33 @@
 - name: minio
   image: docker.io/minio/minio:RELEASE.2024-07-04T14-25-45Z
   command:
-    - minio
-    - server
-    - ./data
+    - /bin/bash
+    - "-c"
+    - |
+      echo "Starting minio"
+      printenv
+      minio server ./data &
+      mc config host add lunchpail http://localhost:9000 $MINIO_ROOT_USER $MINIO_ROOT_PASSWORD --api s3v4
```

Comment:
Do we need any comment (mostly for us) to explain why we picked `--api s3v4`?

Reply:
we need to use `--api` otherwise `mc` will probe the given endpoint to auto-detect the api version... but we know the server isn't ready yet. s3v4 is just the stable api version (there were only ever two stable s3 api versions)